src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  list-style-type: none;
+
+  &__star {
+    margin-left: 4px;
+
+    height: 16px;
+    width: 16px;
+
+    background-image: $star;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: $star-active;
+  }
```

Comment:
You need to create a universal variant for 1,2,3,4,5 starsline. 
Try it. Thanks)